simulation_server/src/main/kotlin/projectcows/Application.kt

```diff
@@ -0,0 +1,22 @@
+package projectcows
+
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import projectcows.plugins.*
+import io.ktor.server.plugins.contentnegotiation.*
+import io.ktor.serialization.gson.*
+import java.text.DateFormat
+
+fun main() {
+    embeddedServer(Netty, port = 8069, host = "0.0.0.0") {
```

Comment:
Would be really nice to load port from an environment file, but I know it's a bit tricky in Java/Kotlin so it's okay if you don't bother.